kouncil-frontend/src/app/brokers/broker.backend.service.ts

```diff
@@ -12,11 +12,11 @@ export class BrokerBackendService implements BrokerService {
 
   constructor(private http: HttpClient) { }
 
-  getBrokerConfig(id: string): Observable<BrokerConfig[]> {
-    return this.http.get<BrokerConfig[]>('/api/configs/' + id);
+  getBrokerConfig(serverId: string, id: string): Observable<BrokerConfig[]> {
+    return this.http.get<BrokerConfig[]>(`/api/configs/${id}?serverId=${serverId}`);
```

Comment:
To jest zdaje się mało po bożemu sposób na query paramsy w angularze. Powinny raczej lecieć przez obiekt HttpParams

Reply:
poprawiłem